scripts/gitlab.js

```diff
@@ -1 +0,0 @@
ch((function(t){var n=t.name;if(!!n.match(/^v1\.\d+$/)){var r=parseInt(n.match(/^v1\.(\d+)$/)[1],10);e.versions.push(r),e.versions.sort(),e.maxVersion=Math.max(r,e.maxVersion),e.minVersion=Math.min(r,e.minVersion),e.versionData[r]={branch:t,color:"transparent"}}}))})).then((function(){return r.GitlabBranches.getBranches("^master").then((function(t){var n=t[0];e.versionData[n.name]={branch:n,color:"transparent"}}))})).then((function(){window.monar.versionData=e,window.monar.versionData.versions.forEach((function(t,n){e.versionData[t].color=i.Colors.getGradientColor("#F00","#FF0",Math.min(Math.max(1/(e.versions.length-1)*n,0),1))}))}))},e.fetchMoreDetails=function(){var e;if(!(null===(e=window.monar)||void 0===e?void 0:e.versionData))return Promise.reject();for(var t=new o.Deferred,n=[],r=[],i=function(e,t){n.push(a.GitlabPipelines.getPipelines({ref:t.branch.name}).then((function(t){return{version:e,pipeline:t[0]}}))),r.push(s.GitlabTags.getTags("^"+t.branch.name).then((function(t){return{version:e,tag:t[0]}})))},l=0,c=Object.entries(window.monar.versionData.versionData);l<c.length;l++){var u=c[l];i(u[0],u[1])}return Promise.all(n).then((function(e){return e.forEach((function(e){window.monar.versionData.versionData[e.version].pipeline=e.pipeline}))})).then((function(){return Promise.all(r)})).then((function(e){return e.forEach((function(e){window.monar.versionData.versionData[e.version].tag=e.tag}))})).then((function(){return t.resolve()})),t.promise}}(t.BranchingVersions||(t.BranchingVersions={}))},function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0});var r=n(1),o=n(0);!function(e){e.getBranches=function(e){void 0===e&&(e="");var t=r.getProjectId(),n=new o.Deferred;return $.ajax("/api/v4/projects/"+t+"/repository/branches?search="+e).then((function(e){n.resolve(e)})).catch(n.reject),n.promise}}(t.GitlabBranches||(t.GitlabBranches={}))},function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0});var r=n(1),o=n(0);!function(e){e.getPipelineSchedules=function(){var e=r.getProjectId(),t=new o.Deferred;return $.ajax("/api/v4/projects/"+e+"/pipeline_schedules").then((function(e){t.resolve(e)})).catch(t.reject),t.promise},e.getPipelineSchedule=function(e){var t=r.getProjectId(),n=new o.Deferred;return $.ajax("/api/v4/projects/"+t+"/pipeline_schedules/"+e).then((function(e){n.resolve(e)})).catch(n.reject),n.promise},e.getPipelines=function(e){void 0===e&&(e={});var t=r.getProjectId(),n=new o.Deferred;return $.ajax({url:"/api/v4/projects/"+t+"/pipelines",data:e}).then((function(e){n.resolve(e)})).catch(n.reject),n.promise}}(t.GitlabPipelines||(t.GitlabPipelines={}))},function(e,t,n){"use strict";Object.defineProperty(t,"__esModule",{value:!0});var r=n(1),o=n(0);!function(e){e.getTags=function(e){void 0===e&&(e="");var t=r.getProjectId(),n=new o.Deferred;return $.ajax("/api/v4/projects/"+t+"/repository/tags?search="+e).then((function(e){n.resolve(e)})).catch(n.reject),n.promise}}(t.GitlabTags||(t.GitlabTags={}))}]);
```

Comment:
You also need to commit new regenerated scripts or you will break the script for everyone using it. Use `npm run build:prod` and commit result